omnigibson/prims/entity_prim.py

```diff
@@ -21,7 +21,8 @@
 m = create_module_macros(module_path=__file__)
 
 # Default sleep threshold for all objects -- see https://docs.omniverse.nvidia.com/extensions/latest/ext_physics/simulation-control/physics-settings.html?highlight=sleep#sleeping
-m.DEFAULT_SLEEP_THRESHOLD = 0.001
+# Mass-normalized kinetic energy threshold below which an actor may go to sleep
+m.DEFAULT_SLEEP_THRESHOLD = 0.00005
```

Comment:
with this threshold do objects still go to sleep? I'm curious what the performance implications are

Reply:
They do! I found this from an isaac sim source file, seems to be the default. I tested both of these in `house_single_floor` with a Fetch and with object states & transition rules on, got similar fps out of them. I think it was something like 27 vs 27.3.

This was originally a problem because when I teleport a tablespoon on top of a bowl and drop it, it would freeze mid-air, falling asleep even when it seems like it should keep moving. This smaller numbers helps it fall normally and still gets it to sleep once it settles.